src/applications/686c-674/tests/e2e/fixtures/add-child-report-divorce.json

```diff
@@ -91,15 +91,6 @@
       "birthDate": "1991-05-04",
       "spouseIncome": false
     },
-    "veteranInformation": {
-      "fullName": {
-        "first": "Greg",
-        "middle": "A",
-        "last": "Anderson"
-      },
-      "ssn": "796121200",
-      "birthDate": "1933-05-04"
```

Comment:
I thought `ssnLastFour` and `vaFileLastFour` were prefilled and saved under `veteranInformation`?